Core/NoOpRunner.Core/NoOpRunner.cs

```diff
@@ -32,39 +32,36 @@ public NoOpRunner(IConnectionManager connectionManager)
 
             /* SHAPE FACTORY DESIGN PATTERN */
             ShapeFactory shapeFactory = new ShapeFactory();
-            BaseShape shape1 = shapeFactory.GetShape(Shape.Square, 5, 5);
-            BaseShape shape2 = shapeFactory.GetShape(Shape.Circle, 10, 5);
-            BaseShape shape3 = shapeFactory.GetShape(Shape.Rectangle, 15, 5);
+            BaseShape shape1 = shapeFactory.GetShape(Shape.HealthCrystal, 1, 15);
+            BaseShape shape2 = shapeFactory.GetShape(Shape.DamageCrystal, 5, 15);
+            BaseShape shape3 = shapeFactory.GetShape(Shape.Saw, 8, 15);
```

Comment:
All of these currently render to look like simple platforms. Either add sprites for them, or leave that to @gta-6 and make it so sprites aren't added for these (just render them in different colors)

Reply:
Yes, I will leave it for @gta-6

Reply:
I have left this implementation for @gta-6 . I have added code to find which shape is currently being in queue to be painted (GameWindowRenderer.cs)